components/post_view/floating_timestamp/floating_timestamp.jsx

```diff
@@ -38,18 +40,18 @@ export default class FloatingTimestamp extends React.PureComponent {
         );
 
         let className = 'post-list__timestamp';
-        if (this.props.isScrolling) {
+        if (isScrolling) {
             className += ' scrolling';
         }
 
-        if (this.props.isRhsPost) {
+        if (isRhsPost) {
             className += ' rhs';
         }
 
         return (
             <div
                 className={className}
-                style={this.props.stylesOverride}
+                style={toastPresent ? {top: '50px'} : null}
```

Comment:
0/5 maybe we can drive this by className.

Reply:
i am not a fan of using classes if they are not reusable. It is hard to clean up with the amount of css we already have. I am gonna change to class though.